### PR TITLE
UF-10552: Add readme

### PR DESCRIPTION
**Pull request by cheezi0747**
## Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] Removed feature
- [ ] Code style update (formatting etc.)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes

## Does this PR introduce a breaking change?
- [ ] Yes (I have stepped the version number accordingly)
- [x] No
      
## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly (if applicable).
- [x] I have added/updated tests to cover my changes (if applicable).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,123 @@
...
+
+Ensure that these services are running and properly configured before starting this microservice.
+
+## API Documentation
+
+Access the API documentation via Swagger UI:
+
+- **Swagger UI:** [http://localhost:8080/api-docs](http://localhost:8080/api-docs)
+
+Alternatively, refer to the `openapi.yml` file located in the project's root directory for the OpenAPI specification.
```
Comment (josc04) [inline README.md:56]: Remove?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,123 @@
...
+
+## Usage
+
+### API Endpoints
+
+Refer to the [API Documentation](#api-documentation) for detailed information on available endpoints.
```
Comment (josc04) [inline README.md:62]: Change to "See [API Documentation](#api-documentation) for detailed information on available endpoints."?

File: README.md [hunk 3]
```diff
@@ -1,13 +1,123 @@
...
+
+### Example Request
+
+```bash
+curl -X GET http://localhost:8080/api/resource
```
Comment (josc04) [inline README.md:67]: Change to valid endpoint?

File: README.md (hunk 2 above)
Comment (cheezi0747) [reply README.md:62]: No.

File: README.md (hunk 1 above)
Comment (cheezi0747) [reply README.md]: All services have the openapi.yml file available somewhere. Why not refer to this so a potential user can easily access it without starting the service? I update the path. But removing this entirely is counterproductive.

Comment (josc04) [reply README.md]: Ok, then change text - as it is stated right now it's more of a "do this (point to swagger) or do that". The best case is to both point to swagger (which is done in this readme) and also (if available - which I couldn't see that is was) point to the yaml inside the service aswell.

If both possibilities can be fullfilled, then yes - that's the best path to go.

**Review by AndreSoderlund — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (josc04) [reply README.md:62]: Jag skriver på svenska för att undå språkmissförstånd. För mej är meningen "Refer to the..." mer en uppmaning till den som skriver readme:n att referera till dokumentationen, inte en uppmaning för läsaren att se api-dokumentationen för detaljerad info. 

Men, det är en personlig åsikt att byta till Refer till See, båda funkar säkert. Ang att PR fanns för påseende tidigare betyder ju inte att saker är satta i sten bara för att PR:en är godkänd, eller?

**Pull request merged**